packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -34,6 +34,56 @@ typedef ScrollableWidgetBuilder = Widget Function(
   ScrollController scrollController,
 );
 
+class DraggableScrollableSheetController {
+  DraggableScrollableSheetController();
+
+  late _DraggableScrollableSheetScrollController _scrollController;
+
+  void _attach(_DraggableScrollableSheetScrollController scrollController) {
+    _scrollController = scrollController;
+  }
+
+  void reset() {
```

Comment:
We probably want to ensure any ongoing animations (`goBallistic` or `anmiateTo`) are cancelled here, but I think this is a bug currently existing in prod (`...Actuator.reset()`) so it could be left to a later PR?

Also, this reset logic should be shared with the actuator's reset logic (eg `shouldReset` and the controller should both call the same private internal function) to avoid code duplication.